### PR TITLE
Readme: Clarify the refactor task

### PR DESCRIPTION
**Pull request by avivrosenberg**
Updated the first tasks description to clarify that we want candidates to focus on design issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,9 +103,13 @@ in use when testing the ships functions, not during normal operations.
 
 Tasks for challenge completion:
 
-1. Refactor the code in a way that will make implementing the above requirements as easy and
-   straightforward as possible. Make sure you get the exact same output from the program as you did
+1. Refactor the code in a way that will make implementing all of the above requirements (both new and original) as easy and
+   straightforward as possible. Make sure you get the same output from the program as you did
    prior to the refactor.
```
Comment (SimonLdj) [inline README.md]: Are you sure we can require same output? Implementing CrowsNest will most definitely change the output.

**Comment by avivrosenberg** [comment]
Yeah but it says you should refactor, check output and only then add the
new features.
Maybe we don't need to require this, it's mainly to help them check
themselves.

**Comment by SimonLdj** [comment]
I think that's not clear enough, people have asked me about that.
I think we should make it more clear (as only requirement for step 1), or remove it.

**Comment by avivrosenberg** [comment]
Ok, I'll fix it soon.

**Comment by avivrosenberg** [comment]
OK, check it out now. Let me know if there are other issues.

**Pull request merged**